admin/sql/updates/2018-06-12-create-and-modify-indexes-and-constraints.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+
+-- Drop indexes and recreate them to remove UNIQUE constraint
```

Comment:
Remove? is this comment correct?

Reply:
[CREATE UNIQUE INDEX cluster_id_ndx_recording_cluster ON recording_cluster (cluster_id);](https://github.com/metabrainz/messybrainz-server/blob/11c9542ce9251699a023e432320c66cf22c8440d/admin/sql/create_indexes.sql#L10)
Here I've removed the `UNIQUE` constraint on the `index` (similar for `artist_credit_cluster` and `release_cluster` table). Otherwise we can't represent clusters like: 

|cluster_id | recording_gid | updated|
|-- | -- | --|
|6d49962f-fb7b-489f-8656-811c2c8f11fb | 6d49962f-fb7b-489f-8656-811c2c8f11fb |2018-05-27 10:04:52.308466+00|
|6d49962f-fb7b-489f-8656-811c2c8f11fb | 4befe14b-629b-4af6-927e-e940441c5c0c |2018-05-27 10:04:52.308466+00|